pystac/summaries.py

```diff
@@ -61,7 +61,7 @@ def from_dict(cls, d: Dict[str, Any], typ: Type[T] = Any) -> "RangeSummary[T]":
 FIELDS_JSON_URL = "https://cdn.jsdelivr.net/npm/@radiantearth/stac-fields/fields.json"
 
 FIELDS_JSON_LOCAL_PATH = os.path.join(os.path.dirname(__file__), "resources",
-                                      "fields-normalized.json")
```

Comment:
fields-normalized.json is the better choice for non-JS components as all dependencies have been resolved...